### PR TITLE
feat: add history item

### PR DESCRIPTION
**Pull request by PainterPuppets**
image

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -18,6 +18,7 @@
     "classnames": "2.3.2",
     "eslint": "8.49.0",
     "eslint-config-next": "13.4.19",
+    "moment": "2.29.4",
```
Comment (Keith-CY) [inline package.json]: moment.js is not maintained anymore, day.js(https://day.js.org/) can be used

Ref: https://github.com/moment/moment#project-status

File: app/components/history/HistoryItem.tsx [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+      })),
+    };
+  })();
+
+  return (
+    <div
+      {...attrs}
+      className={`${
+        className ?? ""
+      } w-full flex flex-col bg-lighter-color p-[12px] rounded-lg text-sm`}
```
Comment (Keith-CY) [inline app/components/history/HistoryItem.tsx]:
```suggestion
      } w-full flex flex-col bg-lighter-color p-3 rounded-lg text-sm`}
```

**Review by Keith-CY — APPROVED** [review]

**Comment by Keith-CY** [comment]
It can be merged when you feel it's ready

**Comment by PainterPuppets** [comment]
> It can be merged when you feel it's ready

Might want to merge this pr first: https://github.com/Magickbase/sudt-management-ui/pull/6

**Pull request merged**